NightscoutServiceKit/RemoteCommands/Notifications/BolusRemoteNotification.swift

```diff
@@ -0,0 +1,45 @@
+//
```

Comment:
These are the Codable models used for Nightscout. They conform to the RemoteNotification protocol, a NightscoutServiceKit protocol, which allows for these notifications to convert to RemoteCommands, something Loop will be familiar with.